src/components/ListInput.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
+import useList from '../hooks/useList'
+import Button from './Button'
+
+interface Props {
+  items: string[]
+  onChange: (items: string[]) => void
+}
+
+export default function ListInput(props: Props) {
+  const { value, handleChange, handleSave, handleReset, isDirty } = useList(
+    props
+  )
+
+  return (
+    <Wrapper>
+      <Textarea rows={20} value={value} onChange={handleChange} />
+      <Button disabled={!isDirty} onClick={handleReset}>
```

Comment:
Now I want to know what `isDirty` means 🤣

Reply:
Maybe `hasUnsavedChanged` would be more appropriate 😅